.circleci/config.yml

```diff
@@ -34,18 +34,19 @@ commands:
           echo 'Changes detected:'
           echo $changes
 
-          if [[ $changes ]]; then
-            echo 'export ROOT_CHANGE=true' >> $BASH_ENV
+          if [[ $changes ]] || [ $CIRCLE_BRANCH == "master" ]; then
```

Comment:
We need to run the build against all packages when in master, otherwise the visual regression baseline images won't get set 👍🏼